server/config/routes/contacts.js

```diff
@@ -0,0 +1,15 @@
+import express from 'express';
+import { saveNewContact,
+         updateContactById,
+         getContactById,
+         updateContactDoNotCallById } from '../../controllers/contacts';
+// add updateContactInvalidNumberById
+const router = express.Router();
+
+router.route('/').post(saveNewContact);
+router.route('/:id').get(getContactById);
+router.route('/:id').put(updateContactById);
+router.route('/:id').patch(updateContactDoNotCallById);
```

Comment:
I'd say we should reserve patch for soft deletes since we've been using that on other resources too. A put with do_not_call or is_invalid_number params set should be enough for updating contacts appropriately.